### ISSUE TITLE
Axis is reversed with GDAL 3

### ISSUE THREAD

**Issue by kensongzhu** [smellman/ogr2poly#3]
Hi, 

Thanks for the repo, I performed the script in a box with GDAL 3+, later on when I applied the poly file then I realized that the axis is reversed somehow,  Found this issue [https://github.com/OSGeo/gdal/issues/1546](https://github.com/OSGeo/gdal/issues/1546) . 

So it works as expect when I set

```python
wgsSRS.ImportFromEPSG(4326)
wgsSRS.SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER)
```

 Maybe better to check if the current gdal version in os is 2 or 3 , is PR welcome?

**Comment by smellman** [comment]
Thank you your comment.
PR is welcome but I fixes it.

**Comment by ThomasG77** [comment]
Can be closed IMHO because of #5 already merged as it manages both GDAL 2 and 3 versions
